test/e2e-tests.sh

```diff
@@ -48,10 +49,31 @@ TEST_YAML_IGNORES=${TEST_YAML_IGNORES:-""}
 # test for example "s2i"
 TEST_TASKRUN_IGNORES=${TEST_TASKRUN_IGNORES:-""}
 
+# Define this variable if you want to run all tests and not just the modified one.
+TEST_RUN_ALL_TESTS=${TEST_RUN_ALL_TESTS:-""}
+
+
+# Main branch to detect the file changes against to for which test to run
+MAIN_BRANCH=master
+
 set -ex
 set -o pipefail
 
+
+if [[ -n ${TEST_RUN_ALL_TESTS} ]];then
+    all_tests=$(/bin/ls -1d */tests)
```

Comment:
why /bin/ls? did you mean `\ls` ?

Reply:
`/usr/bin/env ls` maybe ? :stuck_out_tongue_closed_eyes: :joy_cat:

Reply:
what I mean was to use `\` - https://www.tldp.org/LDP/GNU-Linux-Tools-Summary/html/c1195.htm#:~:text=Use%20a%20%E2%80%9C%5C%E2%80%9D%20(backslash,run%20it%20without%20the%20alias.&text=The%20backslash%20escape%20character%20can,command%20to%20override%20any%20aliases.

Reply:
I guess we could, i like better the /usr/bin/env approach since less magic (and non-obvious) than the backslash solution (fyi neat, on zsh `=` is a even better shortcut :D)

Reply:
to be explicit i think the recommended way is `command`  e.g. `command ls -al ...`